src/Cli/dotnet/commands/InstallingWorkloadCommand.cs

```diff
@@ -242,6 +243,20 @@ protected void UpdateWorkloadManifests(WorkloadHistoryRecorder recorder, ITransa
                 return;
             }
 
+            if (resolvedWorkloadSetVersion?.Contains('@') == true)
+            {
+                var searchVersionsCommand = new WorkloadSearchVersionsCommand(
+                     Parser.Instance.Parse("dotnet workload search version " + resolvedWorkloadSetVersion),
```

Comment:
If we have to create an instance of another command class, I think it's better if we don't generate a command line and parse it.  Rather, we should have constructor parameters or class properties that let us pass in the data we need.